Dizzy/Screens/ReserveTable/ReserveTableVC.swift

```diff
@@ -0,0 +1,234 @@
+        setupCloseButton()
+        numberOfPeopleTextField.keyboardType = .numberPad
+    }
+    
+    private func setupTitleLabel() {
+        titleLabel.text = "Reserve a table".localized
+        titleLabel.textAlignment = .center
+        titleLabel.textColor = .primeryPurple
+    }
+    
+    private func setupMainmainStackView() {
+        mainStackView.alignment = .fill
+        mainStackView.axis = .vertical
+        mainStackView.spacing = Metrics.doublePadding
+    }
+    
+    private func setupButtonsStackView() {
+        buttonsStackView.alignment = .fill
+        buttonsStackView.axis = .horizontal
+        buttonsStackView.spacing = Metrics.doublePadding
+    }
+    
+    private func setupSendButton() {
+        sendButton.setTitle("SEND".localized, for: .normal)
+        sendButton.titleLabel?.font = Fonts.h10(weight: .bold)
+        sendButton.setTitleColor(.white, for: .normal)
+        sendButton.layer.cornerRadius = signInCornerRadius
+        sendButton.backgroundColor = signInButtonBackgroundColor
+        sendButton.addTarget(self, action: #selector(onSendPressed), for: .touchUpInside )
+    }
+    
+    private func setupCloseButton() {
+        closeButton.setImage(Images.whiteBackButton(), for: .normal)
+        closeButton.addTarget(self, action: #selector(onCloasePressed), for: .touchUpInside )
+    }
+    
+    private func setup(_ button: UIButton, with buttonType: ButtonType) {
+        button.tag = buttonType.rawValue
+        button.setTitle(buttonType.title, for: .normal)
+        button.setTitleColor(.blue, for: .normal)
+        button.addTarget(self, action: #selector(buttonPressed(_:)), for: .touchUpInside)
+    }
+    
+    private func setupPlaceNameLabel() {
+        placeNameLabel.text = viewModel.placeName
+        placeNameLabel.textAlignment = .center
+        placeNameLabel.textColor = .white
+        placeNameLabel.font = Fonts.h1(weight: .bold)
+    }
+    
+    private func setupCommentsTextView() {
+        commentsTextView.font = Fonts.h10()
+        commentsTextView.textContainer.maximumNumberOfLines = 2
+        commentsTextView.textContainer.lineBreakMode = .byWordWrapping
+        commentsTextView.setBorder(borderColor: .lightGray)
+    }
+    
+    private func layoutViews() {
+        mainStackView.snp.makeConstraints { make in
+            make.top.equalToSuperview().offset(Metrics.padding)
+            make.leading.equalToSuperview().offset(Metrics.doublePadding)
+            make.trailing.equalToSuperview().inset(Metrics.doublePadding)
+        }
+        
+        tonightButton.snp.makeConstraints { make in
+            make.width.equalTo(tomorrowButton)
+            make.width.equalTo(otherButton)
+        }
+        
+        commentsTextView.snp.makeConstraints { make in
+            make.height.equalTo(Metrics.doublePadding * 3)
+        }
+        
+        closeButton.snp.makeConstraints { make in
+            make.top.equalToSuperview().inset(Metrics.doublePadding * 2)
```

Comment:
dont use `*` here, you can set a constant or create new variable on `Metrics`

Reply:
Done